CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [1.4.2]
+
+- fix: Use BehaviorSubject for _onAuthStateChangedController [#116](https://github.com/supabase/gotrue-dart/pull/116)
```

Comment:
I think it's better to state the new behavior instead of the way we implemented it. I doubt everyone knows this class.